nautobot/docs/models/dcim/interfaceredundancygroup.md

```diff
@@ -0,0 +1,21 @@
+# Interface Redundancy Groups
+
++++ 1.6.0
+
+Interface Redundancy Groups represent groups of interfaces that share a single virtual address. This model is intended to represent redundancy protocols such as HSRP or VRRP that allow multiple devices to provide a fault-tolerant default gateway for a network.
```

Comment:
I am lacking the language, but I think we should be clear that it is for any interface that is considered redundant, regardless if there is a protocol tying it together. As an example, my two uplinks may just run bgp, but I consider it in my interface redundancy strategy.

Reply:
Thanks @itdependsnetworks 

I added a note to call that out